sw/device/tests/rstmgr_cpu_info_test.c

```diff
@@ -154,8 +161,8 @@ bool test_main(void) {
     LOG_INFO("Booting for the first time, setting wdog");
 
     global_error_cnt = 0;
-    uint32_t bark_cycles = aon_timer_testutils_get_aon_cycles_from_us(200);
-    uint32_t bite_cycles = aon_timer_testutils_get_aon_cycles_from_us(200);
+    uint32_t bark_cycles = aon_timer_testutils_get_aon_cycles_from_us(100);
```

Comment:
just did this to slightly speed up the test...it could probably be made even shorter.